Calling/src/app/utils/localStorage.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const localStorageAvailable = typeof Storage !== 'undefined';
+
+export enum LocalStorageKeys {
```

Comment:
We should merge some logic in Calling/src/app/theming/SwitchableFluentThemeProvider.tsx as it also has local storage access but doesn't use this LocalStorageKeys enum

Reply:
Created as work item 
https://skype.visualstudio.com/SPOOL/_workitems/edit/2693380